src/twisted/conch/test/keydata.py

```diff
@@ -11,21 +11,42 @@
 from twisted.python.compat import long, _b64decodebytes as decodebytes
 
 RSAData = {
```

Comment:
Can we add RSAData2048, instead of replacing this?

We should still support 1024bit keys, so I think that the tests should continue to run with 1024.

Is just that some interoperability tests will now run with 2048 as the external system is no longer happy with 2048